src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -136,6 +136,9 @@ export default function RequestedAppointmentDetailsPage() {
   const isCCRequest =
     appointment.vaos.appointmentType === APPOINTMENT_TYPES.ccRequest;
   const provider = appointment.preferredCommunityCareProviders?.[0];
+  const apptDetails = message
```

Comment:
FYI @jenniemc this will impact your ticket about handling messages for community care requests. You might have to tweak this code, assuming this PR makes it in first.